examples/step-75/doc/results.dox

```diff
@@ -0,0 +1,101 @@
+debug mode, your terminal output should look like this:
+@code
+Running with Trilinos on 4 MPI rank(s)...
+Calculating transformation matrices...
+Cycle 0:
+   Number of active cells:       3072
+     by partition:               768 768 768 768
+   Number of degrees of freedom: 12545
+     by partition:               3201 3104 3136 3104
+   Number of constraints:        542
+     by partition:               165 74 138 165
+   Frequencies of poly. degrees: 2:3072
+   Solved in 5 iterations.
+
+
++---------------------------------------------+------------+------------+
+| Total wallclock time elapsed since start    |      7.94s |            |
+|                                             |            |            |
+| Section                         | no. calls |  wall time | % of total |
++---------------------------------+-----------+------------+------------+
+| calculate transformation        |         1 |     0.544s |       6.8% |
+| compute indicators              |         1 |     0.373s |       4.7% |
+| initialize grid                 |         1 |     0.773s |       9.7% |
+| output results                  |         1 |     0.107s |       1.3% |
+| setup system                    |         1 |     0.988s |        12% |
+| solve system                    |         1 |      5.12s |        64% |
++---------------------------------+-----------+------------+------------+
+
+
+Cycle 1:
+   Number of active cells:       3351
+     by partition:               875 761 843 872
+   Number of degrees of freedom: 18228
+     by partition:               4535 4735 4543 4415
+   Number of constraints:        1202
+     by partition:               303 290 326 283
+   Frequencies of poly. degrees: 2:2522 3:829
+   Solved in 5 iterations.
+
+
++---------------------------------------------+------------+------------+
+| Total wallclock time elapsed since start    |      8.47s |            |
+|                                             |            |            |
+| Section                         | no. calls |  wall time | % of total |
++---------------------------------+-----------+------------+------------+
+| adapt resolution                |         1 |     0.689s |       8.1% |
+| compute indicators              |         1 |     0.454s |       5.4% |
+| output results                  |         1 |      0.11s |       1.3% |
+| setup system                    |         1 |     0.708s |       8.4% |
+| solve system                    |         1 |       6.5s |        77% |
++---------------------------------+-----------+------------+------------+
+
+
+Cycle 2:
+   Number of active cells:       3651
+     by partition:               958 818 933 942
+   Number of degrees of freedom: 26519
+     by partition:               6651 6822 6563 6483
+   Number of constraints:        3013
+     by partition:               801 701 790 721
+   Frequencies of poly. degrees: 2:2385 3:758 4:508
+   Solved in 6 iterations.
+...
```

Comment:
Could you also output the info of the last cycle. Is this debug or release mode. For the final results, let's run it with release mode (on modern CPU architecture).

Reply:
Sounds good.

Reply:
I've updated the terminal output. With the recent changes, the number of solver iterations fluctuates between 7 and 11 in this application.